lib/internal/Magento/Framework/View/Element/UiComponent/Context.php

```diff
@@ -116,6 +123,7 @@ public function __construct(
         UrlInterface $urlBuilder,
         Processor $processor,
         UiComponentFactory $uiComponentFactory,
+        AuthorizationInterface $authorization,
```

Comment:
Please add new dependency to constructor as the last optional parameter (backward compatibility)

Reply:
Please see "Adding constructor parameters" section for details https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#php